index.html

```diff
@@ -4,28 +4,28 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
-    <link rel="icon" href="/favicon.ico" />
+    <link rel="icon" href="%VITE_APP_URL%/favicon.ico" />
```

Comment:
@Haviles04 I'm looking at you, shame! You hardcoded all these productions urls, hahaha. @itsalaidbacklife we should get rid of these and switch to this env method. @jerryh-jr will need it in his unhead PR anyway.

Reply:
Actually, Mal did it here, https://github.com/cuttle-cards/cuttle/commit/7826a8327d784359a8344d18812fd93cb3f54edd, but no one fixed it hahahaha.

Reply:
C'mon, bro, read the docs https://vite.dev/guide/env-and-mode.html#html-env-replacement 😏

Reply:
I mean touché but given that we have 4 different tools for rendering dynamic html between Vue, vite, sails and ejs, it's not always clear _which_ docs to read when seeing a new templating syntax 🤷

Reply:
*puts hands up* it wasn't me!